### PR TITLE
Vp662 what is your job 1.1

### PR DESCRIPTION
**Pull request by Eteroa123**
## I do solemnly swear that I have:
- [ *] Run `npm test` and all the tests pass.
- [ *] Run `npm run lint` and there are no warnings.
- [ *] Not decreased the overall test coverage?
- [ *] Included the Jira ID and description in the PR title

## Proposed Changes? 🤔
1. Added a "What is you job title " input field 
2. Saved the data from person detail form to show on the user profile

## Screenshots 📷
 
Jobtitle

### CODE REVIEW

File: components/Person/PersonDetailForm.js [hunk 1]
```diff
@@ -567,8 +581,8 @@ PersonDetailForm.propTypes = {
     twitter: PropTypes.string,
     website: PropTypes.string,
     pronoun: PropTypes.object,
-    imgUrl: PropTypes.string,
-    imgUrlSm: PropTypes.string,
+    job: PropTypes.string,
+    imgUrl: PropTypes.any,
```
Comment (avowkind) [inline components/Person/PersonDetailForm.js]: it seems 
 imgUrlSm: PropTypes.string,
has been removed - I think this should still be included

File: server/api/person/person.js [hunk 2]
```diff
@@ -24,6 +24,7 @@ const personSchema = new Schema({
   facebook: { type: 'String' },
   twitter: { type: 'String' },
   education: { type: 'String' },
+  job: { type: 'String', default: '' },
```
Comment (avowkind) [inline server/api/person/person.js]: as Job is optional it does not require a default: ''   use the same style as for education.

File: components/Person/PersonDetailForm.js (hunk 1 above)
Comment (Eteroa123) [reply components/Person/PersonDetailForm.js]: Ive added back in now ! Thanks heaps for the feedback

File: server/api/person/person.js (hunk 2 above)
Comment (Eteroa123) [reply server/api/person/person.js]: Oh bless ive taken it out now !!! Thanks heaps

**Pull request merged**